playbook/app/pb_kits/playbook/pb_table/_table.tsx

```diff
@@ -20,6 +20,7 @@ type TableProps = {
   singleLine?: boolean,
   size?: "sm" | "md" | "lg",
   sticky?: boolean,
+  striped?: boolean,
```

Comment:
Hey Carlos! I'd double-check with Colin on this prop name, I seem to remember it was supposed to be called 'zebra-striped'

Reply:
awesome!